tests/makeotfexe_test.py

```diff
@@ -707,6 +707,19 @@ def test_negative_internal_leading_bug1227():
             b'win.ascent + win.descent < unitsPerEm') in output
 
 
+def test_goadb_multiple_unicodes_bug1273():
+    input_filename = "bug1273/font.pfa"
+    ttx_filename = "bug1273.ttx"
+    goadb_filename = 'bug1273/GLyphOrderAndAliasDB'
```

Comment:
The "L" in "GLyph" is capitalized here but not in the filename above. That's working on MacOS because its filesystem is case-insensitive but won't work in other contexts.